doc/rst/developer/bestPractices/buildingdocs.rst

```diff
@@ -193,6 +207,16 @@ heading the file is under.
      - Primers
      - doc/rst/meta/primers/index.rst + test/release/examples/primers/\*.chpl
 
+.. list-table:: **Use doxygen on .cpp files to generate html**
```

Comment:
actually we use doxygen and breathe -- maybe write it as "Use doxygen on .cpp files and then breathe and sphinx to generate html"

Reply:
Done.